filebeat/harvester/log_file.go

```diff
@@ -167,9 +165,6 @@ func (r *LogFile) wait() {
 }
 
 func (r *LogFile) Close() {
-	// Make sure reader is only closed once
-	r.singleClose.Do(func() {
-		close(r.done)
-		// Note: File reader is not closed here because that leads to race conditions
-	})
+	close(r.done)
```

Comment:
why remove the `sync.Once` here? Can we guarantee only one go-routine calling Close just once now?